### PR TITLE
Testflight

### PR DESCRIPTION
**Pull request by PRRugebregt**
Second TestFlight version. 

Added new features : 
- Helpview for first time users. 
- Profile button left corner 
- Sorting and filtering operations on list
- Toggle switch between online and local fee results 

Fixed bugs : 
- Initial loading would not occur. 
- Couldn't delete fees 
- Dark mode would show weird colours. Light mode is now forced.

### CODE REVIEW

**Comment by PRRugebregt** [comment]
Looks good to me!

File: Podfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ platform :ios, '13.0'
 target 'Wage' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
+  inhibit_all_warnings!
```
Comment (PRRugebregt) [inline Podfile:8]: Yeah this was annoying.

File: Wage/Extensions/NotificationExtensions.swift [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+//  Wage
+//
+//  Created by Patrick Rugebregt on 20/02/2022.
+//
+
+import Foundation
+
+extension Notification.Name {
+    static let shareUser: Notification.Name = Notification.Name("shareUser")
+}
```
Comment (PRRugebregt) [inline Wage/Extensions/NotificationExtensions.swift]: We don't use notifications anymore. Please update
```suggestion
} Remove notifications
```

**Review by PRRugebregt — COMMENTED** [review]
Please fix the notifications

**Review by patturik — APPROVED** [review]
Looking good now. Will fix notifications later

**Pull request merged**